velox/core/PlanNode.h

```diff
@@ -803,6 +803,25 @@ class GroupIdNode : public PlanNode {
       std::string groupIdName,
       PlanNodePtr source);
 
+  /// @param id Plan node ID.
```

Comment:
Can we still wrap the old constructor in a macro? The implementation would need to be moved to the .h file for that old constructor.

Reply:
Moving the constructor code over involves moving functions getGroupIdOutputType and getGroupingSets also which seemed cumbersome.
I've added a note about the function deprecation.